tests/json_api.rs

```diff
@@ -397,3 +397,29 @@ fn json_request_fails_when_not_enabled() {
 
   assert_eq!(response.status(), StatusCode::NOT_ACCEPTABLE);
 }
+
+#[test]
+fn get_block() {
+  let rpc_server = test_bitcoincore_rpc::spawn();
+
+  rpc_server.mine_blocks(1);
+
+  let response =
+    TestServer::spawn_with_args(&rpc_server, &["--enable-json-api"]).json_request("/block/0");
+
+  assert_eq!(response.status(), StatusCode::OK);
+
+  let block_json: BlockJson = serde_json::from_str(&response.text().unwrap()).unwrap();
+
+  assert_eq!(
+    block_json,
+    BlockJson {
+      hash: String::from("000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f"),
+      target: String::from("0000000000000000000000000000000000000000000000000000ffff00000000"),
```

Comment:
This target looks backwards to me. It should have one bits on the left side, since it's a number that the block hash must be less than.

Reply:
We should fix it in both places, both here and on the non-json block page.